uber/templates/registration/index.html

```diff
@@ -1,88 +1,124 @@
 {% extends "base-admin.html" %}
 {% block title %}Attendee Admin{% endblock %}
 {% block content %}
-<script type="text/javascript">
-    var $highlit = null;
-    function highlight($tr) {
-        if( $highlit )
-            $highlit.css("border", "1px solid black");
-
-        $highlit = $tr.children("td").css("border-top", "3px solid green").css("border-bottom", "3px solid green");
-        $highlit.filter(":first").css("border-left", "3px solid green");
-        $highlit.filter(":last").css("border-right", "3px solid green");
-
-        if( $tr.find("select").val() == "{{ c.AGE_UNKNOWN }}" )
-            $tr.find("select").parents("td").css("border", "3px solid red");
-        if( $tr.find("td:nth(3)").text() == "no" )
-            $tr.find("td:nth(3)").css("border", "3px solid red");
-
-        var badge_num = $tr.find(":text").val();
-        if( badge_num && !badge_num.match(/^\d+$/) )
-            $tr.find(":text").parents("td").css("border", "3px solid red");
-    }
-
-    function recordMPointUsage() {
-        $.post("record_old_mpoint_exchange", {
-                    badge_num:  $("#badge_num").val(),
-                    amount:     $("#amount").val(),
-                    csrf_token: csrf_token
-                }, function(json) {
-                    $("#message").html(json.message);
-                    if( json.success ) {
-                        $("#badge_num,#amount").val("");
-                        $("#badge_num").focus();
-                        $("#message").append(' &nbsp; ').append(
-                            $('<a href="#">Undo</a>').click(function(){
-                                undoMPointUsage(json.id);
-                                return false;
-                            }));
-                    }
-                }, "json");
-    }
-    function undoMPointUsage(id) {
-        $.post("undo_mpoint_exchange", {"id": id, csrf_token: csrf_token}, function(s) {
-            $("#message").html(s);
-        });
-    }
-
-    function selectGroup(id, attendee) {
-        selectedGroup = $("#group_" + attendee +" option[value='" + id + "']")
-        if (selectedGroup.val()) {
-            selectedGroup.prop('selected',true);
-            return true
-        } else {
-            alert("Invalid group code!")
-        }
-    }
 
-    $(function(){
-        $("#message").ajaxError(function(){
-            showTop("Oh noes - the web server is down or something!!!!!");
-        });
-
-        $("#badge_num,#amount").keypress(function(e){
-            if( (e.keyCode || e.which) == 13 )  // ENTER
-                recordMPointUsage();
```

Comment:
All of the above deleted Javascript code was no longer being used on this page anyway.  The "highlighting" code hasn't been called since the "Quick Check In" dialog was implemented, and the mpoint exchange form was moved to the ``/registration/merch`` page last year.